plugins/scaffolder-react/src/secrets/SecretsContext.tsx

```diff
@@ -21,21 +21,25 @@ import React, {
   PropsWithChildren,
 } from 'react';
 
-type SecretsContextContents = {
+/**
+ * The contents of the {@link SecretsContext}.
+ * @public
+ */
+export type SecretsContextContents = {
   secrets: Record<string, string>;
   setSecrets: React.Dispatch<React.SetStateAction<Record<string, string>>>;
 };
 
 /**
- * The actual context object.
+ * The context to hold the Secrets.
+ * @public
  */
 export const SecretsContext = createContext<SecretsContextContents | undefined>(
   undefined,
 );
```

Comment:
We'll need to introduce a versioned context here, e.g. https://github.com/backstage/backstage/blob/4570cea6c20538b4470e93d232eefb44dc0330f7/plugins/search-react/src/context/SearchContext.tsx#L65-L67